test/tbb/test_task_arena.cpp

```diff
@@ -2038,3 +2039,22 @@ TEST_CASE("is_inside_task in arena::execute") {
     });
 }
 #endif //__TBB_PREVIEW_TASK_GROUP_EXTENSIONS
+#endif
+
+//! The test for a regression when producer of affinity tasks and consumer left
+//! the arena and no one can proceed the tasks
+//! \brief \ref regression
+TEST_CASE("abandon afifinity tasks deadlock") {
+    const int num_arenas = 16;
+    std::vector<tbb::task_arena> arenas(num_arenas);
+    for (int trail = 0; trail < 1000000; trail) {
```

Comment:
Maybe ```trail++```

Reply:
It was incorrect search and replace... Overall, the test does not reproduce the supposed issue and I will remove it completely.